models/migrations/v156.go

```diff
@@ -50,15 +52,10 @@ func fixPublisherIDforTagReleases(x *xorm.Engine) error {
 	sess := x.NewSession()
 	defer sess.Close()
 
-	if err := sess.Begin(); err != nil {
-		return err
-	}
-
 	var (
 		gitRepoCache = make(map[int64]*git.Repository)
```

Comment:
We may need to limit the size of this or... perhaps even better just order the releases by repository ID